core-bundle/src/Resources/contao/elements/ContentDownloads.php

```diff
@@ -266,14 +265,14 @@ protected function compile()
 		{
 			default:
 			case 'name_asc':
-				uksort($files, static function (): int
+				uksort($files, static function ($a, $b): int
```

Comment:
@fritzmg How can this ever have worked in Contao 4.9?

Reply:
The sort function would have always returned `0` so I think the array would not have been sorted at all.

Reply:
So we should definitely backport this. 😄